src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -325,6 +328,19 @@ function MdAutocomplete ($$mdSvgRegistry) {
       // will hold the actual tabindex.
       element.attr('tabindex', '-1');
 
+      var disableVirtualRepeat = $mdAutocompleteCtrl.scope.disableVirtualRepeat;
```

Comment:
It looks like `mdAutocompleteCtrl` isn't defined here (outside of the actual template string). `!!attr. disableVirtualRepeat` should work.

Reply:
:+1: